app/api/auth/thirdweb/authentication.ts

```
@@ -27,12 +27,12 @@ export async function login(payload: VerifyLoginPayloadParams) {
 
 export const getJwtContext: () => Promise<JwtContext> = async () => {
   const jwt = cookies().get("jwt");
-
+  
   if (!jwt?.value) {
-    throw new Error(`Failed to fetch JWT context, jwt.value is not defined`);
+    throw new Error(`Failed to fetch JWT context, jwt is not defined`);
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)